airbyte-integrations/connectors/source-snowflake/src/main/java/io.airbyte.integrations.source.snowflake/SnowflakeSource.java

```diff
@@ -113,4 +107,10 @@ private JsonNode buildUsernamePasswordConfig(final JsonNode config, final String
     return Jsons.jsonNode(configBuilder.build());
   }
 
+  @Override
```

Comment:
Unless I am missing something, I don't see any functional changes in this file. Only that `getExcludedInternalNameSpaces()` got moved from line 84 to line 111. Can this file be excluded from this PR?

Reply:
Yeah, this was mentioned in the GL review from me too but it appears to be some kind of weird bug since the changes here already existed and I see what Andrii sees in my own IDE too: https://github.com/airbytehq/airbyte/pull/15832/files#r955972633